hs_core/tasks.py

```diff
@@ -91,7 +93,15 @@ def on_failure(self, exc_info, send_failed_event=True, return_ok=False):
 
 
 class HydroshareTask(Task):
+    '''Custom Celery Task configured for Hydroshare
+    https://docs.celeryq.dev/en/v4.4.7/userguide/tasks.html?#automatic-retry-for-known-exceptions
+    '''
     Request = HydroshareRequest
+    autoretry_for = (Exception, KeyError)
```

Comment:
these properties aren't that different from the default, but it won't hurt to have a jitter (so that jobs don't all execute at the same time) and a backoff (so that external APIs like geoconnex don't throttle our requests)